src/config.ts

```diff
@@ -78,6 +77,8 @@ export const fhirConfig: FhirConfig = {
             },
         },
     },
+
+    defaultRetryRequestInSeconds: 60 * 15,

```

Comment:
why do we need this? I don't think that customers want to customize this value. 

I'd rather start with a static non-configurable value. Also, later on when we improve the rate-limiting strategy this value can be computed by the server based on the current load of export requests.

Reply:
It wasn't too hard to make this value configurable and depending on what the user picks for their persistence layer, I could foresee them wanting to change this value. They might want the retry value to be longer or shorter than 15 minutes.

Reply:
I think we should be more careful about adding config values. It seems easy, but they are part of the interface so once added we cannot take them back. 

I still don't see the case for customers wanting to customize this value. We can add it later if it is a feature request.

It's worth noting that the `Retry-After` header in 429 responses is just guidance. The client is not forced to honor it and can actually retry at whatever interval he chooses. So it's very  unlikely to be a blocker for any customer.

If we do end up having this as a config value, I don't think it should be a top level key. The name `defaultRetryRequestInSeconds` is too generic. We'd need to think better about the hierarchy of this kind of system config values. (Maybe `system.export.defaultRetryAfterSeconds` ?)

Reply:
My 2 cents here is export is effectively some sort of operation that a customer may not want to support so I think it fits better in the `profile` section (similar to bundles)

to Nestor's other point of 'is it worth it' from a config perspective? I can see it both ways it is a trivial part of the spec that the server doesn't even have to recognize BUT it is a part of the spec. I like Nestor's point of we can always add it in but we cannot remove it. my vote is to remove it for now in favor of a constant

Reply:
That's a good point about not being able to remove the config once it's been added. I'll remove it for now.